### ISSUE TITLE
Would you like an extra maintainer on this project? 

### ISSUE THREAD

**Issue by Faunsce** [caffeine-overload/bandinchina#99]
I understand that you've put alot of work into this project but as the issues and pr's pile up im sure its a bit stressing. just wanted to see if youd like an extra hand. #FreeHongKong

**Comment by caffeine-overload** [comment]
Yeah, it would probably be for the best. Also pinging @Medallyon since he has contributed a lot. 

I think we also should create a "guidance document" on what kind of action gets a company put on the blacklist or whitelist

**Comment by Faunsce** [comment]
That seems more like a section in a CONTRIBUTING.md

**Comment by caffeine-overload** [comment]
Yeah, I will write a provisional one
